contract-addresses/dependencies.ts

```diff
@@ -1,5 +1,735 @@
+      'rariPool91FeiPCVDeposit',
+      'rariPool9FeiPCVDeposit'
+    ]
+  },
+  feiTribeLBPSwapper: {
+    contractDependencies: ['core', 'pcvEquityMinter']
+  },
+  optimisticMinter: {
+    contractDependencies: ['core', 'optimisticTimelock']
+  },
+  pcvEquityMinter: {
+    contractDependencies: ['core', 'collateralizationOracleWrapper', 'feiTribeLBPSwapper']
+  },
+  pcvGuardian: {
+    contractDependencies: [
+      'core',
+      'guardian',
+      'feiDAOTimelock',
+      'daiPSM',
+      'compoundEthPCVDeposit',
+      'aaveEthPCVDeposit',
+      'ethReserveStabilizer'
+    ]
+  },
+  proxyAdmin: {
+    contractDependencies: [
+      'feiDAOTimelock',
+      'aaveTribeIncentivesController',
+      'tribalChief',
+      'collateralizationOracleWrapper'
+    ] // NOTE this is slightly misleading as proxy admin needs to update admin to new timelock
+  },
+  ratioPCVController: {
+    contractDependencies: ['core']
+  },
+  tribe: {
+    contractDependencies: [
+      'core',
+      'rariPool8Tribe',
+      'feiDAO',
+      'tribeRariDAO',
+      'erc20Dripper',
+      'aaveTribeIncentivesController',
+      'tribeMinter',
+      'tribeReserveStabilizer'
+    ]
+  },
+  tribeMinter: {
+    contractDependencies: ['core', 'tribeReserveStabilizer', 'tribe', 'erc20Dripper', 'feiDAOTimelock' /* Owner */]
+  },
+  feiDAO: {
+    contractDependencies: ['feiDAOTimelock', 'tribe']
+  },
+  feiDAOTimelock: {
+    contractDependencies: [
+      'core',
+      'feiDAO',
+      'fei',
+      'proxyAdmin',
+      'creamDepositWrapper',
+      'aaveTribeIncentivesController',
+      'tribeMinter',
+      'pcvGuardian'
+    ]
+  },
+  guardian: {
+    contractDependencies: ['core', 'collateralizationOracleGuardian', 'pcvGuardian'] // TODO do we want to document everything the guardian can affect. I think this should only reflect guardian-exclusive actions
+  },
+  optimisticMultisig: {
+    contractDependencies: ['optimisticTimelock']
+  },
+  optimisticTimelock: {
+    contractDependencies: [
+      'core',
+      'rewardsDistributorAdmin',
+      'tribalChiefSync',
+      'rariPool8Comptroller',
+      'optimisticMultisig',
+      'optimisticMinter',
+      'tribalChief',
+      'collateralizationOracle',
+      'collateralizationOracleWrapper',
+      'staticPcvDepositWrapper2'
+    ]
+  },
+  rariTimelock: {
+    contractDependencies: ['tribeRariDAO']
+  },
+  tribeRariDAO: {
+    contractDependencies: ['rariTimelock', 'tribe']
+  },
+  aaveEthPCVDripController: {
+    contractDependencies: ['core', 'fei', 'aaveEthPCVDeposit', 'ethReserveStabilizer']
+  },
+  bondingCurve: {
+    contractDependencies: ['core', 'fei', 'aaveEthPCVDeposit', 'compoundEthPCVDeposit', 'chainlinkEthUsdOracleWrapper']
+  },
+  compoundEthPCVDripController: {
+    contractDependencies: ['core', 'fei', 'compoundEthPCVDeposit', 'ethReserveStabilizer']
+  },
+  daiBondingCurve: {
+    contractDependencies: ['core', 'fei', 'compoundDaiPCVDeposit', 'chainlinkDaiUsdOracleWrapper']
```

Comment:
because this is deprecated, can't we just remove?

Reply:
not deprecated yet